src/Components/TestExplorer.fs

```diff
@@ -1412,10 +1439,18 @@ module Interactions =
                 |> Option.map (fun p -> p.kind = TestRunProfileKind.Debug)
                 |> Option.defaultValue false
 
+            let hasIncludeFilter =
```

Comment:
This is actually a small unrelated fix.
Project-level items were constructing test filter clauses for all their top-level children.
But, that meant it wouldn't discover top-level tests if they weren't in the explorer yet.

The new behavior is to use no filter if just the top-level project is being run.
I can split this out if you'd prefer